apkid/rules/apk/packers.yara

```diff
@@ -218,6 +218,24 @@ rule dexprotector_b : packer
     not dexprotector
 }
 
+rule dexpro_a : packer
+{
+  meta:
+    description = "DexProtector for AIDE"
+    url         = "https://github.com/rednaga/APKiD/issues/197"
+    sample      = "e113be26d90fe2cb287009345139fba0c550a67b15c3022eb5dc13aa0eb8235a"
+    author      = "Eduardo Novella"
+
+  strings:
+    // pkgname  = mph.dexprotect.a
```

Comment:
What does this mean?

Reply:
the package name of the packer just for extra info